scripts/release/check_ios_appstore_verify.py

```
@@ -0,0 +1,536 @@
+
+Results = List[Tuple[bool, str, str]]
+
+
+def _read_png_dimensions(path: pathlib.Path) -> Tuple[int, int]:
+    """Read width/height from PNG IHDR chunk (no external deps)."""
+    with open(path, "rb") as fh:
+        sig = fh.read(8)
+        if sig[:4] != b"\x89PNG":
+            raise ValueError(f"Not a PNG file: {path}")
+        # IHDR is always first chunk after signature.
+        _length = fh.read(4)
```

Flake8 (7.3.0)

[error] local variable '_length' is assigned to but never used

(F841)